pkg/config/loader.go

```
@@ -0,0 +1,928 @@
+
+	if len(atmosFoundFilePaths) == 0 {
+		return nil, fmt.Errorf("Failed to find config files in path '%s'", gitRootDir)
+	}
+	cl.AtmosConfigPaths = append(cl.AtmosConfigPaths, gitRootDir)
+	return atmosFoundFilePaths, err
+}
+
+// loadConfigsFromPath loads configuration files from the specified path.
+// It returns a boolean indicating if any configs were found and loaded,
+// a slice of loaded config paths, and an error if any.
+func (cl *ConfigLoader) loadConfigsFromPath(path string) (bool, []string, error) {
```

golangci-lint (1.62.2)

func `(*ConfigLoader).loadConfigsFromPath` is unused

(unused)